sros2/sros2/keystore/_keystore.py

```diff
@@ -14,28 +14,26 @@
 # limitations under the License.
 
 import os
+import pathlib
 
 from cryptography import x509
 
 import lxml
 
+from sros2 import _utilities
+import sros2.errors
 from sros2.policy import get_transport_default, get_transport_schema
 
-from . import _utilities
-
 
 _KS_ENCLAVES = 'enclaves'
 _KS_PUBLIC = 'public'
 _KS_PRIVATE = 'private'
 _DEFAULT_COMMON_NAME = 'sros2testCA'
 
 
-def create_keystore(keystore_path):
-    if not is_valid_keystore(keystore_path):
-        print('creating keystore: %s' % keystore_path)
-    else:
-        print('keystore already exists: %s' % keystore_path)
-        return
+def create_keystore(keystore_path: pathlib.Path) -> None:
+    if is_valid_keystore(keystore_path):
+        raise sros2.errors.KeystoreExistsError(keystore_path)
 
     os.makedirs(keystore_path, exist_ok=True)
```

Comment:
```suggestion
    keystore_path.mkdir(exist_ok=True)
```

Reply:
Good catch, I fixed that whole chunk there.